src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -267,45 +268,47 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter {
 
         protected virtual bool RequestIsCacheable(ActionExecutingContext filterContext) {
 
+            var itemDescriptor = string.Format("{0} (Controller: {1}, Action: {2})", filterContext.RequestContext.HttpContext.Request.RawUrl, filterContext.ActionDescriptor.ControllerDescriptor.ControllerName, filterContext.ActionDescriptor.ActionName);

```

Comment:
`{0}` is the raw URL, which will include the query string, eg: `/OrchardLocal?hello=1 (Controller: Item, Action: Display)`.

This value has already been logged at https://github.com/paynecrl97/Orchard/blob/5390/delay_output_cache_key_creation/src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs#L83, so I made the assumption that I was safe to log the full query string.

I could probably add in Tenant, Headers, Theme, Culture & Auth though (I *think* that those are all the params that are included in the OOTB cache key).